inst/ms/aggreCAT.qmd

```diff
@@ -596,62 +625,81 @@ Below we demonstrate the application of `ArMean` on a single claim
+in the form of a [tibble]{.class}:
 
 ```{r}
 #| label: focal-claim-ArMean
 #| prompt: true
+#| message: false
 focal_claims %>% 
-  filter(paper_id == "108") %>%
+  dplyr::filter(paper_id == "108") %>%
   AverageWAgg(type = "ArMean")
 ```
 
 ```{r}
 #| label: fig-ArMean
-#| fig-cap: "`ArMean()` uses the Estimates (shown in colour) from each participant to compute the mean. We illustrate this using a single claim `zttvyg` for a subset of 5 out of 25 participants from the `data_ratings` dataset. Note that the data representations in this figure are for explanatory purposes only, the data in the actual aggregation is tidy, with long form structure and format."
+#| fig-cap: "ArMean with `AverageWAgg()` uses the Estimates (shown in colour) from each participant to compute the mean. We illustrate this using a single claim `zttvyg` for a subset of 5 out of 25 participants from the `data_ratings` dataset. Note that the data representations in this figure are for explanatory purposes only, the data in the actual aggregation is tidy, with long form structure and format."
 #| results: hold
 #| echo: false
-knitr::include_graphics(path = "images/paste-80B1C5CB.png")
+knitr::include_graphics(path = "images/ArMean.png")
 ```
 
-::: {#aggWorkflow .callout}
-#### Box 1: Aggregation Workflow Blueprint
-
--   wrapper functions and the 'type' argument
-
--   default arg structure for each wrapper fun
-
-    -   percent toggle
-
-    -   placeholder
-
-    -   name
-
-    -   What else?
+::: {.callout}
+:::{#aggWorkflow}
+### Box 1: Aggregation Workflow Blueprint{.unnumbered}
 
--   COmputation:
+#### Argument Structure and Expected Form
 
-    -   Weighting Functions
-
-    -   Note that not all functions use a separately defined weighting
-        function -- for simpler weight computations, these are defined
-        in-function rather than being modularised [See Table
-        1](#table1).
-
-Each aggregation function follow a general workflow whereby the primary
-dataset `data_ratings`, parsed to the `expert_judgements` argument, is
-first pre-processed by `pre_process_judgements()`, subsequently the
-aggregation method is applied using `dplyr::summarise()`, and then
-finally the aggregated data is parsed to `postprocess_judgements()`.
-This general aggregation method workflow is seen best in `ArMean()`:
+Each aggregation *wrapper* function takes the following arguments:
+`expert_judgements`, `type`, `name`, `placeholder` and `percent_toggle`:
 
 ```{r}
-#| label: blah
-AverageWAgg
+#| prompt: true
+args(AverageWAgg)
 ```
 
+The aggregation *method* to be applied by the aggregation *function*, is
+specified by the `type` argument, defaulting to `ArMean` in the above example. The resultant
+`tibble` of Confidence Scores includes the `name` of the aggregation
+method applied, defaulting to the `type` argument, but this can be
+overridden by the user if they supply a non-`NULL` value to `name`.
+
+Aggregation functions expect the judgements supplied in
```

Comment:
Not quite true. Percentages are our "assumed" default fit, but it could be things like counts too. You set `percent_toggle` to `TRUE` convert percentages to probabilities (this syntax is probably confusing I guess, maybe we should reverse it). You leave it set to `FALSE` if you're supplying probabilities or counts etc